src/atropos.less

```diff
@@ -72,12 +72,15 @@
     opacity: 1 !important;
   }
 }
-[data-atropos-offset] {
+::slotted(img) {
```

Comment:
This is wrong , it must be not an image but elements with `[data-atropos-offset]`

Reply:
plus chaning it to slotted only will break other plugins